backend/inspirehep/records/api/mixins.py

```diff
@@ -553,3 +554,70 @@ def get_modified_institutions_uuids(self):
         pids_changed = set.symmetric_difference(set(pids_latest), set(pids_previous))
 
         return list(self.get_records_ids_by_pids(list(pids_changed)))
+
+
+class ExperimentPapersMixin:
```

Comment:
These relation mixin classes are starting to be exactly the same. We should think of a way to refactor them and extract the common logic. I'll add a separate task for that though